include/aws/common/zero.h

```diff
@@ -0,0 +1,98 @@
+#ifndef AWS_COMMON_ZERO_H
+#define AWS_COMMON_ZERO_H
+
+/*
+ * Copyright 2010-2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Should this just be 2019?

Reply:
I genuinely have no idea. I copy/pasted "2010-2018" from another header and thought I was real cool for updating 18 to 19